kshape/core.py

```diff
@@ -158,6 +158,11 @@ def _extract_shape(idx, x, j, cur_center):
 
 
 def _kshape(x, k,n_init=1, max_iter=100, n_jobs = 1, random_state=None):
+    """
+    Runs multiple initializations of kshape and returns the best in terms of ShapeBasedDistance. 
+    max_iter is the maximum number of iterations within one run, 
+    n_jobs is the number of jobs: 1 is non-parallel version, if -1, all CPUs are used
+    """
```

Comment:
`_kshape` is internal, I think you wanted to add it to `kshape()` instead.

Reply:
Good call. Fixed in 
https://github.com/Mic92/kshape/pull/8/commits/3427048d8b0d985b706650eebc1597ac09339c9 .e